PWGJE/Tasks/PhiInJetsJE.cxx

```diff
@@ -0,0 +1,412 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file PhiInJets.cxx
+/// \brief Reconstruction of Phi yield through track-track Minv correlations for resonance hadrochemistry analysis.
+///
+///
+/// \author Adrian Fereydon Nassirpour <adrian.fereydon.nassirpour@cern.ch>, Jimun Lee <jimun.lee@cern.ch>, Bong-Hwi Lim <bong-hwi.lim@cern.ch>, Sawan Sawan <sawan.sawan@cern.ch>
+
+#include <TLorentzVector.h>
+#include <TVector2.h>
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+
+#include "Common/Core/RecoDecay.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/TrackSelectionDefaults.h"
+#include "Common/Core/trackUtilities.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/Multiplicity.h"
+//#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/PIDResponse.h"
+
+#include "ReconstructionDataFormats/Track.h"
+
+#include "PWGHF/Core/PDG.h"
+
+#include "PWGJE/Core/FastJetUtilities.h"
+#include "PWGJE/Core/JetDerivedDataUtilities.h"
+#include "PWGJE/DataModel/Jet.h"
+
+#include "PWGLF/DataModel/LFResonanceTables.h"
+
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+#include "Framework/runDataProcessing.h"
+
+
+struct PhiInJetsJE {
+  SliceCache cache;
+  //Preslice<aod::Tracks> perCollision = aod::track::collisionId;
+  HistogramRegistry JEhistos{"JEhistos", {}, OutputObjHandlingPolicy::AnalysisObject};
+  HistogramRegistry LFhistos{"LFhistos", {}, OutputObjHandlingPolicy::AnalysisObject};
+
+
+  Configurable<std::string> c_eventSelections{"c_eventSelections", "sel8", "choose event selection"};
+  Configurable<std::string> c_trackSelections{"c_trackSelections", "globalTracks", "set track selections"};
```

Comment:
Underscores are only allowed in names of namespaces.

Reply:
fixed